archive/android_bootldr.ksy

```diff
@@ -0,0 +1,36 @@
+meta:
+  id: android_bootldr
+  title: Qualcomm Snapdragon (MSM) bootloader.img format
+  file-extension: img
+  tags:
+    - archive
+    - android
+  license: CC0-1.0
+  endian: le
+doc-ref: https://android.googlesource.com/device/lge/hammerhead/+/7618a7/releasetools.py
```

Comment:
I suggest adding a `doc` with some resources where one can get sample files:

- https://developers.google.com/android/images
- https://tmp.androidfilehost.com/?w=files&flid=300713#:~:text=bootloader.img

I like the second link particularly because you don't have to download 1+ GB .zip archive just to obtain a single `bootloader.img` file a with few MBs from it (which is the case when you use the Google website; you also need to pick the right device there because only several of them use this "archive" format for `bootloader.img`). However, it is a bit inconvenient to work with it in the Web IDE because of its relatively large size (~36 MB). It also uses different partitions than the _Hammerhead's `bootloader.img`_ that you link in `/doc-ref`, but the overall "archive" structure is the same.

---

Anyway, you should also add at least a few sentences about the format in [the `/doc` key](https://doc.kaitai.io/ksy_style_guide.html#doc), i.e. some general information about the format? What it is used for? What kind of data it stores? What is the main usage field?

The point is for someone who visits the format page in our format gallery to get a basic idea about what kind of the format it is, and not have to search it on Wikipedia or other external websites.

Reply:
I have added some stuff in the description. If you want more, let me know.